contrib/nydusify/cmd/nydusify.go

```diff
@@ -226,6 +226,12 @@ func main() {
 					return fmt.Errorf("--backend-type should be one of %v", possibleBackendTypes)
 				}
 
+				fsVersion := c.String("fs-version")
```

Comment:
Perhaps we can reuse this variable below.

Reply:
Done.